dashboard/client/src/common/ProfilingLink.tsx

```diff
@@ -307,7 +307,7 @@ export const MemoryProfilingButton = ({
   if (!pid || !ip) {
     return <div></div>;
   }
-  const profilerUrl = `/memory_profile?pid=${pid}&ip=${ip}`;
+  const profilerUrl = `memory_profile?pid=${pid}&ip=${ip}`;
```

Comment:
Can you help me understand what this fixes?

Is it the because the ray dashboard server is being hosted on a subpath from root?